docs/application/native/guides/device/system.md

```diff
@@ -159,6 +159,16 @@ The following table lists the consumer IR feature keys.
 |----------------------------------------|--------|------------------------------------------|
 | `http://tizen.org/feature/consumer_ir` | `bool` | The platform returns `true` for this key, if the device supports the Consumer Infrared (CIR) feature. |
 
+The following table lists the content feature keys.
+
+<a name="content"></a>
+**Table: Content feature keys**  
+
+| Key                                    | Type   | Description                              |
+|----------------------------------------|--------|------------------------------------------|
+| `http://tizen.org/feature/content.scanning.others` | `bool` | The platform returns true for this key, if the device supports the media scanning feature for OTHERS type files which are not included in the media content-types such as IMAGE, VIDEO, SOUND or MUSIC. |
```

Comment:
Please change "true" to \`true\` in here and next line.

The platform returns \`true\` for this key,...

Reply:
Done